mediabridge/data_processing/wiki_to_netflix.py

```diff
@@ -190,4 +264,5 @@ def process_data(test=False):
     # Test is true if no argument is passed or if the first argument is not '--prod'.
     test = len(sys.argv) < 2 or sys.argv[1] != "--prod"
     process_data(test=test)
+    test = len(sys.argv) < 2 or sys.argv[1] != "--prod"
```

Comment:
nit: Looks like a duplicate line from a merge? Please remove.

Reply:
Will do thanks for catching that